crates/sui-core/src/authority.rs

```diff
@@ -249,6 +249,9 @@ pub struct AuthorityState {
     /// The checkpoint store
     pub(crate) checkpoints: Option<Arc<Mutex<CheckpointStore>>>,
 
+    /// Stores next expected sequence number from validators that we follow
+    pub(crate) follower_store: Option<Arc<FollowerStore>>,
```

Comment:
Do you anticipate a case where `follower_store` is not enabled?

Reply:
I refactored this to avoid the Option